viewer/config.js

```diff
@@ -500,10 +500,65 @@ exports.configMap = function (section, dSection, d) {
   return map;
 };
 
+function loadCertData () {
+  exports.keyFileLocation = exports.get('keyFile');
+  exports.certFileLocation = exports.get('certFile');
+
+  // eslint-disable-next-line no-useless-catch
+  try {
+    exports.keyFileData = fs.readFileSync(exports.keyFileLocation);
+    exports.certFileData = fs.readFileSync(exports.certFileLocation);
+  } catch (err) {
+    throw err;
+  }
+}
+
 if (exports.isHTTPS()) {
-  exports.keyFileData = fs.readFileSync(exports.get('keyFile'));
-  exports.certFileData = fs.readFileSync(exports.get('certFile'));
+  try {
+    loadCertData();
```

Comment:
@awick we don't need this anymore if we can assume that setServerToReloadCerts is called for each server that sets certs. This is duplicated on line 528.

Reply:
ok